module_utils/helpers.py

```diff
@@ -610,6 +619,48 @@ def patch_clusterwide_config(control_console, new_sections):
     return True, None
 
 
+def get_disabled_instances(console_sock, stateboard):
+    if stateboard:
+        return None, None
+
+    control_console, err = get_control_console_if_started(console_sock)
+    if err is not None:
+        return None, err
+    if not control_console:
+        return None, None
+
+    instances = get_cluster_instances(control_console)
+    return [name for name, info in (instances or {}).items() if info.get('disabled')], None
+
+
+def get_topology_checksum(console_sock, stateboard):
+    if stateboard:
+        return None, None
+
+    control_console, err = get_control_console_if_started(console_sock)
+    if err is not None:
+        return None, err
+    if not control_console:
+        return None, None
+
+    return control_console.eval_res_err('''
+        local digest = require('digest')
+        local confapplier = require('cartridge.confapplier')
+
+        local active_config = confapplier.get_active_config()
+        if active_config == nil then
+            return nil
+        end
+
+        local topology_section = active_config:get_plaintext('topology.yml')
```

Comment:
Are we support versions of Cartridge that have another config format?

Reply:
Yes, it works with Cartridge 2.1.2